pkg/packagekit/updates.jsx

```diff
@@ -845,14 +845,17 @@ class OsUpdates extends React.Component {
                         </button>);
                 }
 
-                if (highest_severity == PK.Enum.INFO_SECURITY)
-                    text = _("Security Updates Available");
-                else if (highest_severity >= PK.Enum.INFO_NORMAL)
-                    text = _("Bug Fix Updates Available");
-                else if (highest_severity >= PK.Enum.INFO_LOW)
-                    text = _("Enhancement Updates Available");
-                else
+                if (highest_severity == PK.Enum.INFO_SECURITY) {
+                    text = cockpit.format(_("$0 Security Updates Available"), num_security_updates);
+                } else if (highest_severity >= PK.Enum.INFO_NORMAL) {
+                    const num_normal = count_updates_of_severity(this.state.updates, PK.Enum.INFO_NORMAL);
+                    text = cockpit.format(_(" $0 Bug Fix Updates Available"), num_normal);
+                } else if (highest_severity >= PK.Enum.INFO_LOW) {
+                    const num_enhancement = count_updates_of_severity(this.state.updates, PK.Enum.INFO_LOW);
+                    text = cockpit.format(_(" $0 Enhancement Updates Available"), num_enhancement);
```

Comment:
Severities are only compared/counted exactly. I. e. if there are no security updates, 1 bug fix, and 23 enhancements, it would still say "1 bug fix update available", and completely disregard the others. It feels a bit wrong to me, but I don't have a better proposal right now. Expanding it to "1 security, 2 bug fix, and 5 enhancement updates available" makes the sentence a bit long. We already went through several iterations of this, and it's always quite a non-trivial discussion.

Honestly I would just keep this particular bit ouf of the front page redesign, as it's really unrelated. If we want to change it, let's do that in a separate PR.

If you want to go ahead with this: This needs to use ngettext, and drop the leading spaces from the strings.

Reply:
I changed it because @garrett on his mockup has exact numbers.
https://garrett.github.io/cockpit-mockweb/navigation2/
I will leave it up on him then.

Reply:
Oh, the mockup was just supposed to show that the updates line goes there. I'm on Silverblue, so I don't see the actual string (due to not having traditional updates), so I just wrote something as a placeholder instead of copying actual text.

I'm fine with keeping it as it is in master for now. We can revisit later.